client/app/components/composites/ListingCardPanel/ListingCardPanel.css

```diff
@@ -2,7 +2,7 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-around;
-  padding: calc(var(--ListingCardPanel_gutterSpacing) / 2);
+  padding: calc(var(--ListingCardPanel_gutterSpacing) / 4);
```

Comment:
Changed this to explicit padding (18px) - managed to do that by removing margins from listing cards in single column layout (mobile)